standard/date_ext.lua

```diff
@@ -123,7 +123,7 @@ end
 ---@param offsetString string?
 ---@return integer # default `0`
 function DateExt.getOffsetSeconds(offsetString)
-	return 0 - tonumber(mw.getContentLanguage():formatDate('U', '1970-01-01T00:00:00' .. (offsetString or '')))
+	return 0 - tonumber(mw.getContentLanguage():formatDate('U', '0000-01-01T00:00:00' .. (offsetString or '')))
 end
```

Comment:
do we still need 1970 here?
(i.e. will `mw.getContentLanguage():formatDate` be adjusted in backend too or not)
cc @FO-nTTaX

Reply:
Should still be 1970 here

Reply:
what Rathoz said, as this function explicitly deals with unixtime timestamps